src/node_debug_options.cc

```diff
@@ -141,7 +141,8 @@ int DebugOptions::port() const {
   int port = port_;
   if (port < 0) {
 #if HAVE_INSPECTOR
-    port = inspector_enabled_ ? default_inspector_port : default_debugger_port;
+    port = (debugger_enabled_ && !inspector_enabled_) ? default_debugger_port
+                                                      : default_inspector_port;
```

Comment:
Can you make this an if statement for legibility?  You could also simplify it to this:
```c++
if (port < 0) {
  port = default_debugger_port;
#if HAVE_INSPECTOR
  if (inspector_enabled_)
    port = default_inspector_port;
#endif  // HAVE_INSPECTOR
}
```